beacon_node/eth1/src/service.rs

```diff
@@ -301,9 +316,22 @@ impl Service {
     pub async fn update(
         &self,
     ) -> Result<(DepositCacheUpdateOutcome, BlockCacheUpdateOutcome), String> {
+        let endpoint = &self.config().endpoint.clone();
```

Comment:
We could save a call to the eth1 node by using `eth_getBlockByNumber` with 'latest' in the params to get the block corresponding to remote head.